docs/ams-schema-spec.bs

```diff
@@ -1,174 +1,1208 @@
 <pre class='metadata'>
-Title: Amsterdam Schema Specification
+Title: Amsterdam Schema Specificatie
 Shortname: ams-schema-spec
-Revision: 1.1.1
+Revision: 2.0.0
 Status: LS
 URL: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html
 Repository: Amsterdam/amsterdam-schema
 Editor: Michiel Trimpe, Gemeente Amsterdam, https://data.amsterdam.nl/
-Abstract: Het Amsterdam Schema is een standaard voor JSON Schema's welke gebruikt kan worden om
-    datasets in detail op machine readable formaat te kunnen omschrijven zodat deze generiek
-    verwerkt en ontsloten kunnen worden.
+Abstract: Het Amsterdam Schema is een op JSON Schema’s gebaseerde standaard (ondersteund door het Amsterdam Meta Schema)
```

Comment:
Prima, moest de zin wel een paar keer lezen. Misschien dit?

Het Amsterdam Schema is een op JSON Schema’s gebaseerde standaard (ondersteund door het Amsterdam Meta Schema) om datasets in een machine leesbaar formaat te beschrijven en op generieke wijze te verwerken en te ontsluiten.

Reply:
ja dat is beter.